src/actions/spring.js

```diff
@@ -0,0 +1,93 @@
+/*
+  The closed-form damped harmonic oscillating spring.
+  Or, spring.
+
+  This is a direct port of Adam Miskiewicz's (@skevy) React Animated
+  PR #15322 https://github.com/facebook/react-native/pull/15322/
+
+  ```
+  spring({
+    mass: 2,
+    damping: 10,
+    stiffness: 100,
+    to: 100
+  }).start();
+  ```
+
+  Adam Miskiewicz:
+  @skevy (twitter.com/skevy, github.com/skevy)
+ */
+import Action from './';
+import { timeSinceLastFrame } from '../framesync';
+
+class Spring extends Action {
+  static defaultProps = {
+    stiffness: 100,
+    damping: 10,
+    mass: 1.0,
+    velocity: 0.0,
+    from: 0.0,
+    to: 0.0,
+    restSpeed: 0.01,
+    restDisplacement: 0.01
+  };
+
+  onStart() {
+    const { velocity, to } = this.props;
+    this.t = 0;
+    this.initialVelocity = velocity / 1000;
```

Comment:
What's an example set of values you were using (before you divided v0 by 1000? I coded the velocity in `Animated` to also be per second...for example I usually get values for velocity in the 0-10 (+/-) px/sec range. This whole function is calculated in per second values (`t` is in seconds, thus `dt` (velocity) is also per second).